### ISSUE TITLE
Config logging not reporting correct version

### ISSUE THREAD

**Issue by tylerbenson** [DataDog/dd-trace-java#3686]
I configured `DD_VERSION` via an environment variable, but it isn't showing up in the config printed out:

```
INFO datadog.trace.agent.core.StatusLogger - DATADOG TRACER CONFIGURATION 
{
    "version": "0.90.0~32708e53ec",
    "os_name": "Linux",
    "os_version": "4.14.255-276-224.499.amzn2.aarch64",
    "architecture": "aarch64",
    "lang": "jvm",
    "lang_version": "11.0.14.1",
    "jvm_vendor": "Amazon.com Inc.",
    "jvm_version": "11.0.14.1+10-LTS",
    "java_class_version": "55.0",
    "http_nonProxyHosts": "null",
    "http_proxyHost": "null",
    "enabled": true,
    "service": "usage-service",
    "agent_url": "http://localhost:8126",
    "agent_error": false,
    "debug": false,
    "analytics_enabled": false,
    "sampling_rules": [
        {},
        {}
    ],
    "priority_sampling_enabled": true,
    "logs_correlation_enabled": true,
    "profiling_enabled": false,
    "appsec_enabled": false,
    "dd_version": "0.90.0~32708e53ec",
    "health_checks_enabled": false,
    "configuration_file": "no config file present",
    "runtime_id": "8d08f87a-4fe4-4974-93fa-00af1ed3f237",
    "logging_settings": {
        "levelInBrackets": false,
        "dateTimeFormat": "'[dd.trace 'yyyy-MM-dd HH:mm:ss:SSS Z']'",
        "logFile": "System.err",
        "configurationFile": "simplelogger.properties",
        "showShortLogName": false,
        "showDateTime": true,
        "showLogName": true,
        "showThreadName": true,
        "defaultLogLevel": "INFO",
        "warnLevelString": "WARN",
        "embedException": false
    },
    "cws_enabled": false,
    "cws_tls_refresh": 5000
}
```

(The correct version does get reported to Datadog on the spans though.)

Notice though that the javaagent version is reported twice...

**Comment by mcculls** [comment]
Thanks @tylerbenson - https://github.com/DataDog/dd-trace-java/pull/3687 should fix the logging

**Comment by tylerbenson** [comment]
Thanks @mcculls!

**Comment by mcculls** [comment]
This change was released in [0.106.0](https://github.com/DataDog/dd-trace-java/releases/tag/v0.106.0)
